### PR TITLE
Add Q/A about Certificates validness after account unsubscribal

### PR DESCRIPTION
**Pull request by san983**
A customer asked about what happens with his valid certificate when he unsubscribes and I think that can be added to the SSL certificate FAQ

If this make sense, I'll ask @itsalyse for checking the phrasing/wording.

## :mag: QA

- Go to http://127.0.0.1:3000/articles/faq-ssl-certificates/
- Check that the new Question "I’m unsubscribing my account, will my current certificate keep valid?" is there.

## :shipit: Verification

- [ ] Check the update section is deployed https://support.dnsimple.com/articles/faq-ssl-certificates/

### CODE REVIEW

File: content/articles/faq-ssl-certificates.markdown [hunk 1]
```diff
@@ -42,6 +42,13 @@ categories:
     An SSL certificate is not a single-time purchase. You need to renew the certificate periodically, sometimes you need to rekey it or you find yourself having issues installing it, and you want to ask for some help. All these tasks are covered by our subscription service.
 
     Additionally, if you are requesting a [Let's Encrypt certificate](/articles/letsencrypt/), we will use our DNS hosting service to automatically provision the DNS records required to validate your certificate, and we will [automatically renew](/articles/letsencrypt/#auto-renewal) the certificate for you before it expires. All these services require a subscription.
+
+1.  #### I'm unsubscribed/ing my account, will my current certificate keep valid?
```
Comment (jacegu) [inline content/articles/faq-ssl-certificates.markdown]: The _"unsubscribed/ing"_ is something that I don't think we have done before. Maybe you could better phrase this as: 
> Will my certificate remain valid after unsubscribing?

File: content/articles/faq-ssl-certificates.markdown [hunk 2]
```diff
@@ -42,6 +42,13 @@ categories:
...
+
+    Yes. As explained in the previous questions, you are not required to have an active subscription to use or install the certificate on your domain/server.
```
Comment (jacegu) [inline content/articles/faq-ssl-certificates.markdown]: Certificates are installed on servers. Don't make customers think hey are installed on domains or we will be flooded with questions about how to do that 😉

Comment (san983) [reply content/articles/faq-ssl-certificates.markdown]: Right! 797dee9a259541d62773e0393843d4c6ba3f0087

File: content/articles/faq-ssl-certificates.markdown (hunk 1 above)
Comment (san983) [reply content/articles/faq-ssl-certificates.markdown]: Sounds better, thanks. 797dee9a259541d62773e0393843d4c6ba3f0087

**Pull request merged**